modular_skyrat/modules/delam_emergency_stop/code/scram.dm

```diff
@@ -333,18 +355,18 @@
 	message_admins("[ADMIN_LOOKUPFLW(user)] just opened the cover of the [src].")
 	investigate_log("[key_name(user)] opened the cover of the [src].", INVESTIGATE_ATMOS)
 
-	confirm_action()
+	confirm_action(user)
 
 /// Confirms with the user that they really want to push the red button. Do it, you won't!
 /obj/machinery/button/delam_scram/proc/confirm_action(mob/user, list/modifiers)
-	if(tgui_alert(usr, "You really sure that you want to push this?", "It looked scarier on HBO.", list("No", "Yes")) != "Yes")
+	if(tgui_alert(usr, "Are you really sure that you want to push this?", "It looked scarier on HBO.", list("No", "Yes")) != "Yes")
 		button_stage = BUTTON_AWAKE
 		visible_message(span_danger("[user] slowly closes the plastic cover of the [src]!"))
 		update_appearance()
 		return
 
 	// Make scary sound and flashing light
-	playsound(src, 'sound/machines/high_tech_confirm.ogg', 50, FALSE, 15, ignore_walls = TRUE, use_reverb = TRUE)
+	playsound(src, 'sound/machines/high_tech_confirm.ogg', 50, FALSE, 7, ignore_walls = TRUE, use_reverb = TRUE, falloff_distance = BUTTON_SOUND_FALLOFF_DISTANCE)
 	button_stage = BUTTON_PUSHED
 	visible_message(span_danger("[user] smashes the [src] with their hand!"))
```

Comment:
You don't actually need a `the` if it's already a lowercase name, the game will add the `the` for you if it's improper, when using the `"[var]"` string interpolation (I think that's the term for it).
```suggestion
	visible_message(span_danger("[user] smashes [src] with their hand!"))
```

Reply:
If it's not improper, because it somehow starts with a capital letter, either make it lower case, or if you can't, append a `"\improper"` at the start of it.

Reply:
I thiiink it's the latter and I just didn't use improper in the name